src/RDF/Editor/WorkflowEditor.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\RDF\Editor;
+
+use EasyRdf\Graph;
+use EasyRdf\Literal;
+use EasyRdf\Resource;
+
+final class WorkflowEditor
```

Comment:
```suggestion
final class WorkflowStatusEditor
```
Small nit but this may be clearer, since "Workflow" could also be interpreted as something specific to RDF/Graphs maybe, while "WorkflowStatus" is a known concept

Reply:
Done in this commit https://github.com/cultuurnet/udb3-backend/pull/1331/commits/8dee494e3be781f53aeb1c4b9b78c21a9700924d